dockstore-client/src/test/resources/YamlVerifyTestDirectory/service-empty-files-parameter/.dockstore.yml

```diff
@@ -0,0 +1,21 @@
+version: 1.2
+service:
+  subclass: DOCKER_COMPOSE
+  authors:
+    - name: Test
+      email: test@test.net
+  description: This is a test
+  publish: true
+  scripts:
+    start: start.sh
+  files:
+  environment:
+    TEST-1:
+      default: testing
+      description: testing testing testing
+  data:
+    test-2:
+      targetDirectory: ~/test
```

Comment:
This would've just been me filling out the template. I just copied a `.dockstore.yml` file I had already created for this file.

However, after reading through the [template](https://docs.dockstore.org/en/stable/assets/templates/services/services.html#full-template-with-explanation-of-all-available-fields), it appears this is not valid, as it requires that the `targetDirectory` is a relative path.

I have corrected this for all `.dockstore.yml` test files that I created.

I have also added this [comment](https://github.com/dockstore/dockstore/issues/5443#issuecomment-1487089867) to this [ticket](https://github.com/dockstore/dockstore/issues/5443) to either determine correct `DockstoreYamlHelper.validateDockstoreYamlProperties()` such that it catches this in the future, or to update the [template](https://docs.dockstore.org/en/stable/assets/templates/services/services.html#full-template-with-explanation-of-all-available-fields) to indicate that absolute paths are acceptable.